ament_pclint/ament_pclint/main.py

```diff
@@ -0,0 +1,372 @@
+#!/usr/bin/env python3
+
+# Copyright 2017 Apex.AI, Inc.
+#
+#
+# This file contains modified code from the following open source projects
+# published under the licenses listed below:
+#
+#
+# # Copyright 2014-2015 Open Source Robotics Foundation, Inc.
+# #
+# # Licensed under the Apache License, Version 2.0 (the "License");
+# # you may not use this file except in compliance with the License.
+# # You may obtain a copy of the License at
+# #
+# #     http://www.apache.org/licenses/LICENSE-2.0
+# #
+# # Unless required by applicable law or agreed to in writing, software
+# # distributed under the License is distributed on an "AS IS" BASIS,
+# # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# # See the License for the specific language governing permissions and
+# # limitations under the License.
+
+import argparse
+import concurrent.futures
+import multiprocessing
+import os
+from shutil import which
+import subprocess
+import sys
+import time
+from xml.etree import ElementTree
+from xml.sax.saxutils import escape
+from xml.sax.saxutils import quoteattr
+
+
+def main(argv=sys.argv[1:]):
+    extensions = ['c', 'cc', 'cpp', 'cxx', 'c++']
+
+    parser = argparse.ArgumentParser(
+        description='Perform static code analysis using pclint.',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument(
+        'paths',
+        nargs='*',
+        default=[os.curdir],
+        help='Files and/or directories to be checked. Directories are searched recursively for '
+             'files ending in one of %s.' %
+             ', '.join(["'.%s'" % e for e in extensions]))
+    parser.add_argument(
+        '--language',
+        help="Force pclint to analyze files as specified language. Supported languages c or cpp")
+    if os.name == 'nt':
+        parser.add_argument(
+            '--pclint-config-dir',
+            nargs=1,
```

Comment:
The default value of `nargs` is 1 so no need to specify it explicitly.

Same below.

Reply:
Done